accounts/keystore/account_cache_test.go

```diff
@@ -382,7 +382,7 @@ func TestUpdatedKeyfileContents(t *testing.T) {
 	}
 
 	// needed so that modTime of `file` is different to its current value after os.WriteFile
-	time.Sleep(time.Second)
+	os.Chtimes(file, time.Now().Add(-time.Second), time.Now().Add(-time.Second))
```

Comment:
Looks a bit odd to me that you are (over and over) setting mtime/atime to `time -1`. If we want them to _change_ shouldn't you set them to e.g. `time-5`, `time-4`, `time-3`, ... etc?

Reply:
I thought that `forceCopyFile` will always update the timestamp to the current time, thats why we can just set the time to -1 sec